manualtest/dbstress/main.go

```diff
@@ -244,6 +244,20 @@ func (ts *testingStorage) Remove(fd storage.FileDesc) error {
 	return ts.Storage.Remove(fd)
 }
 
+func (ts *testingStorage) Reads() uint64 {
+	if s, ok := ts.Storage.(storage.MeteredStorage); ok {
+		return s.Reads()
+	}
+	return 0
+}
+
+func (ts *testingStorage) Writes() uint64 {
+	if s, ok := ts.Storage.(storage.MeteredStorage); ok {
+		return s.Writes()
+	}
+	return 0
+}
```

Comment:
I don't think you need these any more.

Reply:
If the `MeteredStorage` interface will be kept, maybe this will be needed as well in order to print the `ioStats` in the `manualtest/dbstress`. 
After the `db.GetProperty("leveldb.iostats")` call `*testingStorage` is casted to `MeteredStorage`, and in case of success `ioStats` is returned.